src/components/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import logoImg from "../assets/logo.svg";
+import "./Navbar.css";
+import "./global.css";
+import { Link, NavLink } from "react-router-dom";
+
+function getLinkStyle({ isActive }) {
+  return {
+    color: isActive ? "#3692FF" : "#4B5563",
+    textDecoration: isActive ? "none" : "none",
```

Comment:
p4;
요부분은 isActive 상태와 관련없이 모두 'none'이어서 삼항 연산자를 쓰지 않아도 괜찮을 것 같아요!
```suggestion
    textDecoration: "none",
```